### PR TITLE
Added preset for man_made=observatory.

### PR DESCRIPTION
**Pull request by vershwal**
In reference to #4855.
@bhousel Please let me know if I have missed anything.

### CODE REVIEW

**Comment by vershwal** [comment]
The tests are failing. Please suggest how can I fix the tests.

**Comment by Xavier-J-Ortiz** [comment]
@vershwal Hello!

I believe this document could help you a bit more under the https://github.com/openstreetmap/iD/blob/master/CONTRIBUTING.md#adding-or-refining-presets section of the main CONTRIBUTING.md .

It's kind of easy to overlook. But there's a presets readme in that section that should clear things up.

As a final reference, check out this previous closed PR that had a similar issue which I believe that you're running into: https://github.com/openstreetmap/iD/issues/4826#issuecomment-369249797

File: data/presets/presets/man_made/observatory.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "type": [
```
Comment (bhousel) [inline data/presets/presets/man_made/observatory.json]: This should be "terms"

File: data/presets/presets/man_made/observatory.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        "astronomical",
+        "meteorological",
+        "espionage"
```
Comment (bhousel) [inline data/presets/presets/man_made/observatory.json]: I'd remove this word "espionage", it's not really a term related to an [observatory](https://en.wikipedia.org/wiki/Observatory).

File: data/presets/presets/man_made/observatory.json (hunk 1 above)
Comment (bhousel) [reply data/presets/presets/man_made/observatory.json]: By the way this is what is causing the test failures.   "type" doesn't fit here and so the build process will raise an error when it tries to validate the `observatory.json` file against the [preset schema](https://github.com/openstreetmap/iD/blob/master/data/presets/schema/preset.json).

**Comment by vershwal** [comment]
Hey @bhousel !
Thanks for the review. Seems like it's working fine now.

**Comment by bhousel** [comment]
Thanks!

**Pull request merged**